pkg/openshift/templates/data.json.tpl.go

```diff
@@ -0,0 +1,181 @@
+package templates
+
+var DataTemplate = `
+{{define "data"}}
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "mongodb",
+    "annotations": {
+      "description": "Mongodb",
+      "tags": "rhmap,mongodb"
+    }
+  },
+  "objects": [
+    {
+      "kind": "Service",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "data",
+        "labels": {
+          "name": "data",
+          "rhmap/domain": "{{.Domain}}",
+          "rhmap/env": "{{.Env}}",
+          "rhmap/title": "{{.ServiceName}}"
+        }
+      },
+      "spec": {
+        "ports": [
+          {
+            "protocol": "TCP",
+            "port": 27017,
+            "targetPort": 27017,
+            "nodePort": 0
+          }
+        ],
+        "selector": {
+          "name": "mongodb-replica"
+        },
+        "portalIP": "None",
+        "clusterIP": "None",
+        "type": "ClusterIP",
+        "sessionAffinity": "None"
+      }
+    },
+    {
+      "kind": "PersistentVolumeClaim",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "mongodb-claim-1",
+		"labels": {
+          "name": "data",
+          "rhmap/domain": "{{.Domain}}",
+          "rhmap/env": "{{.Env}}",
+          "rhmap/title": "{{.ServiceName}}"
+        }
+      },
+      "spec": {
+        "accessModes": [
+          "ReadWriteOnce"
+        ],
+        "resources": {
+          "requests": {
+			{{if isset .Options "storage"}}  
```

Comment:
Would it be easier to set Options storage to 1Gi by default, and have this as just:
```
"storage": "{{index .Options "storage"}}"
```